_data/snippets.yml

```diff
@@ -114,3 +114,31 @@ enterprise_3_encryption_key_backup: |
   1. Make sure you have appropriate access to the kubernetes cluster: you need credentials for `kubectl` and connection to [travis-api-pod] 
   2. Run `kubectl exec -it [travis-api-pod] cat /app/config/travis.yml |grep -A 2 encryption` (using Travis API pod is recommended). 
   3. Create a backup of the value returned by that command by either writing it down on a piece of paper or storing it on a different computer.
+github_oauth_access_rights: | 
+  When you sign in to [travis-ci.com](https://travis-ci.com) using GitHub for the first time, you will receive a message from GitHub saying:
+  
+  > Travis CI by travis-pro wants to access your [account name] account.
+  
+  and in the repositorie's section it will state: 
+  
+  > This application will be able to read and write all public and private repository data.
+  
+  This is not how Travis CI access yout data, however we can explain it later in the process. 
+  
+  The warning is triggerred due to GitHub OAuth App permissions which Travis CI uses, and due to the available granularity of permission scopes (see GitHub [Scopes for OAuth Apps documentation](https://docs.github.com/en/developers/apps/scopes-for-oauth-apps) ).
+  
```

Comment:
The warning is **triggered** due to GitHub OAuth App permissions which Travis CI uses, and due to the available granularity of permission scopes (see GitHub [Scopes for OAuth Apps documentation](https://docs.github.com/en/developers/apps/scopes-for-oauth-apps) ).

Reply:
The warning **triggers** due to GitHub OAuth App permissions which Travis CI uses, and **also** due to the available granularity of permission scopes (see GitHub [Scopes for OAuth Apps documentation](https://docs.github.com/en/developers/apps/scopes-for-oauth-apps) ).